server/libs/userlibs.js

```diff
@@ -148,4 +149,100 @@ const logOut = function(request) {
     })
 }
 
-module.exports = {register, logIn, updateUser, logOut}
\ No newline at end of file
+const adminDeActivateUser = function(body) {
+    let id = body.uuid;
+    return new Promise((resolve,reject) => {
+        User.findOne({uuid : id}).then((user) => {
+            if (user) {
+                if (user.isAdmin) {
+                    let query = {
+                        deActivate : true,
+                        deletedAt : Date.now(),
+                        deletedBy : user.email
+                    }
+                    User.findOneAndUpdate({uuid : id, deActivate : false}, query, {new : true}).then((user) => {
```

Comment:
use update query and don't need to send whole user in response just a message

Reply:
Done